_assignments/mp1.md

```diff
@@ -0,0 +1,444 @@
+
+You can control the rotation and position of a `GameObject` with the `transform` object variable, an instance of the aptly named [`Transform`](https://docs.unity3d.com/ScriptReference/Transform.html) class.
+Use the functions of this class to rotate the planet system around the $$Y$$-axis.
+
+Keep in mind that the `Update` function runs every frame, but frames often vary in realtime length.
+Thus, using static rotation amounts with the `Transform` class will make the apparent rotation amount depend on framerate.
+This is generally not desirable.
+To use realtime frame times instead, use the [`Time.deltaTime`](https://docs.unity3d.com/ScriptReference/Time-deltaTime.html) variable.
+When used inside the `Update` function, this variable's value is the real time in seconds that elapsed since the last frame.
+See the [Time and Frame Management](https://docs.unity3d.com/Manual/TimeFrameManagement.html) page for more information.
+
+### Break Out
+
+> Set up a button on the controllers to switch the player's position between an external viewing point and the room.
+
+Like all `GameObject`s, the player object has a `Transform`.
+With that in mind, make a script that switches the player between a room and a new external viewing point,
+e.g. a small new plane a moderate distance from the room.
+The player should still start within the room.
+Pressing the button for the first time must move the player to the external viewing point.
+After that, pressing the button must alternate between the two locations.
+
+Remember that the coordinates for the player refer to its center, and the player is 2 units tall.
+Thus, the player controller needs to be 1 unit above the ground.
+
+### Trigger Zone
+
+> Place a suspended sphere in a corner of the room.
+Trigger the sphere to drop when the player moves under the sphere.
+
+You may find the Unity [colliders](https://docs.unity3d.com/Manual/CollidersOverview.html) manual helpful.
+
+By default, colliders register and react with collisions with other `GameObject`s in the physics engine.
+Colliders can also behave like triggers, which fire whenever a `GameObject` intersects a given zone, but do not otherwise alter physics.
+
+Create a new empty `GameObject`, go to `Add Component` → `Physics` → `Box Collider`, and then check the "Is Trigger" option.
+The collider should show up in the scene view as a green wireframe.
+Make sure that no other objects intersect the collider initially!
+
+Add a script to your collider object, and implement the function called when something enters the trigger.
+Again, pay attention to the subtlety between when something **enters** the trigger and when something is **within** the trigger.
+To actually make the sphere fall, enable gravity on the [Rigidbody](https://docs.unity3d.com/ScriptReference/Rigidbody.html), a [physics component](https://docs.unity3d.com/Manual/class-Rigidbody.html), of the sphere.

```

Comment:
I would also want to this video as a reference, https://www.youtube.com/watch?v=bh9ArKrPY8w.